api.py

```diff
@@ -167,6 +167,44 @@ def list_group_members(team_id, group_id, **kwargs):
     return make_get_request('team/{}/group/{}/users'.format(team_id, group_id), **kwargs)
 
 
+def set_owner(team_id, resource_id, **kwargs):
```

Comment:
I see what you mean, but i'd rather `set_owner` be agnostic to whether it's setting the owner to a team or a user.  I may not have a choice here though.